src/Abc.Zebus.MessageDsl/Generator/ProtoGenerator.cs

```diff
@@ -80,7 +80,8 @@ private void WriteEnum(EnumDefinition enumDef)
     private void WriteMessage(MessageDefinition message)
     {
         Writer.WriteLine();
-        Writer.Write("message {0} ", message.Name);
+        var name = string.Join("_", message.ContainingClasses.Select(x => x.NetType).Append(message.Name));
```

Comment:
Wouldn't `ProtoBufType` be better than `NetType` here?

Reply:
Indeed, didnt noticed this prop.